.eslintrc.js

```diff
@@ -30,7 +30,7 @@ module.exports = {
   rules: {
     "react/display-name": "off",
     "react/prop-types": "off",
-    "react/jsx-boolean-value": "off",
+    "react/jsx-boolean-value": "error",
```

Comment:
Are you sure about `error`? When we update the package, we'll have to put some work to fix it immediately

Reply:
real men are not afraid of some work ;)

Reply:
Ok, so thats why i am afraid 😨

Reply:
based